internal/config/config.go

```diff
@@ -72,6 +78,10 @@ func NewDefault() *Config {
 			BaseUrl:   "https://localhost:3333",
 			LogLevel:  "info",
 		},
+		Queue: &queueConfig{
+			AmqpURL:       "http://localhost:5672",
+			ManagementURL: "http://localhost:15672",
```

Comment:
Not a change request. I am trying to remember how AMQP worked, I remember the main URL but what was the management URL for?

Reply:
Is that rabbitmq specific?

Reply:
Maybe.  Actually we don't need now.  I will remove